src/containers/Auth/SignUp/SignUp.js

```diff
@@ -128,47 +128,48 @@ export class SignUp extends Component {
     const subHeader = '“Today a reader, tomorrow a leader”';
     const { state: { error }, props: { isLoading } } = this;
     return (
-      <FormWrapper imageUrl={image} displayImage={true}>
+      <FormWrapper imageUrl={image} displayImage={true} img_id='img-responsive'>
         <FormContainer header={header} subHeader={subHeader}
-          onSubmit={this.handleSubmit}>
-          <div className="signup-container">
+          login_id='login-wrapper' onSubmit={this.handleSubmit}
```

Comment:
Same here @Danielshow. `CamelCase` would be better here

Reply:
Ok